07-statistical-testing.Rmd

```diff
@@ -1 +1,570 @@
+    b. **Comparison to a different value:** $I(var - value) \sim 0$, where $var$ is the measure of interest and $value$ is what we are comparing to. We need to use the `I()` function, to tell the program to calculate the difference between the variable and the comparison value prior to testing. For example, we could test if the proportion of the population that has blue eyes is different from $25%$.
+2. **Two-sided t-test:**
+    a. **Unpaired:**
+        - **2 level grouping variable:** $var \sim groupVar$, where $var$ is the measure of interest and $groupVar$ is a variable with two categories.  For example, we could test if the perfect of the popualtion that has blue eyes is different for men and women. 
+        - **3+ level grouping variable:** $var \sim I(groupVar == level)$, where $var$ is the measure of interest, $groupVar$ is the categorical variable, and $level$ is the category level to isolate. Again we need to use the `I()` function to tell the program to isolate the category before doing the comparison across groups.  For example, we could test if the test scores in one classroom differed from all other classrooms.
+    b. **Paired:** $I(var_1 - var_2) \sim 0$, where $var_1$ is the first variable of interest and $var_2$ is the second variable of interest. We again will have to use the `I()` function to have the program calculate the difference between the two variables before comparing it against $0$.  For example, we could test if test scores on a subject differed between the start and the end of a course.
+
+Additionally, the `na.rm` argument defaults to `FALSE`, which means if any data is missing the t-test will not compute. Throughout this chapter we will always set `na.rm=TRUE`, but before analyzing the survey data, make sure to review the notes provided in Chapter \@ref(c03) to better understand how to handle missing data.  Finally, the `level` argument is $1-\alpha$, or the amount of type 1 error.  The default is $0.95$.
+
+### Examples {#stattest-ttest-examples}
+Let's walk through a few examples using the ANES and RECS data.  See Section \@ref(stattest-setup) above to set up the design objects.
+
+#### Example 1: One-sided t-test {.unnumbered #stattest-ttest-ex1}
+
+ANES asked respondents if they voted early in the 2020 election.  In our data, we've called this variable `EarlyVote2020`.  If we want to see if the proportion of the U.S. voting eligible population that early voted is greater than 0%, we could set up the hypothesis as follows:
+
+- $H_0: p = 0$ where $p$ is the proportion of U.S. voting eligible population that voted early
+- $H_A: p \neq 0$
+
+To conduct this in R, we would then use the `svyttest()` function:
+```{r}
+#| label: stattest-ttest-ex1
+ttest_ex1<-anes_des %>%
+   svyttest(formula=I(EarlyVote2020=="Yes")~0,
+            design=.,
+            na.rm=TRUE)
+
+ttest_ex1
```

Comment:
Opportunity to introduce broom::tidy here. Something like:

```
broom::tidy(ttest_ex1)

> broom::tidy(ttest_ex1)
# A tibble: 1 × 8
  estimate statistic  p.value parameter conf.low conf.high method                         alternative
     <dbl>     <dbl>    <dbl>     <dbl>    <dbl>     <dbl> <chr>                          <chr>      
1    0.948      226. 7.17e-77        50    0.939    0.0440 Design-based one-sample t-test two.sided  
```

Reply:
Something is wrong here and I'm not sure if it's the package. 94.8% of people DIDN'T early vote

````
> anes_des %>%
+   filter(!is.na(EarlyVote2020)) %>%
+   group_by(EarlyVote2020) %>%
+   summarise(
+     pct=survey_prop(proportion=TRUE, vartype = "ci")
+   )
# A tibble: 2 × 4
  EarlyVote2020    pct pct_low pct_upp
  <fct>          <dbl>   <dbl>   <dbl>
1 Yes           0.0525  0.0446  0.0616
2 No            0.948   0.938   0.955 
````